easylinks/src/main/webapp/display_script.js

```diff
@@ -56,7 +56,7 @@ function showPublicLinksInCurrentPage() {
       let row = tableElement.insertRow();
       row.insertCell(0).innerHTML = link.key.id;
       row.insertCell(1).innerHTML = link.propertyMap.shortcut;
-      row.insertCell(2).innerHTML = link.propertyMap.url;
+      addWordBreakToCell(row.insertCell(2), link.propertyMap.url);
```

Comment:
Maybe also change script.js for customized links display? I guess they are the same.

Reply:
Done